docs/api_docs/matlab.rst

```diff
@@ -1,66 +1,153 @@
 
-``data.short_name2 = 1;``
+``list`` returns a list of all parameters available from a client. For example, if we run ``list`` on an instance of ``PropellerMotorControlClient`` called ``prop``, we receive:
 
-``client_object.set_all(data);``
+.. image:: ../_static/api_pics/prop_list.png
 
-will send set messages for all fieldnames in data.
+Get
+^^^^^^
+All MATLAB *get* commands have the format ``client_object.get("client_entry")``.
 
-``client_object.set_verify(’short_name’, value);``
+The ``get`` function returns the value of a single client entry returned by the module through IQUART. The values of “client_entry” are specified by the Short Name 
+column found in the :ref:`Client Table Reference <iquart_client_reference_tables>`.
 
-performs the same function as set, but also performs a get to verify transmission. It will retry up to 10
-times if transmission fails.
+Suppose we want to monitor the module's input voltage. We can do this through the volts entry of the Power Monitor client.
+
+.. image:: ../_static/api_pics/volts_entry.png
+
+Assuming we already have a functioning ``MessageInterface`` called ``com``, we create an instance of the ``PowerMonitorClient`` ``power = PowerMonitorClient('com', com);``. 
+Then, we can request the module's input voltage with ``power.get(‘volts’)``. If everything is configured properly, the module will respond, and the response value returned by the ``get`` function. For example:
+
+.. image:: ../_static/api_pics/get_volts_matlab.png
+
+You can also treat the returned value as a normal parameter, and store it in a variable.
+
+Get All
+^^^^^^^^^^
+All MATLAB get all commands have the format ``client_object.get_all()``.
+
+The ``get_all`` function returns the value of all client entries returned by the module through IQUART. For example, running ``get_all`` on our ``PowerMonitorClient`` returns:
+
+.. image:: ../_static/api_pics/power_mon_get_all.png
+
+You can also treat the returned value as a normal parameter, and store it in a variable.
+
+Set
+^^^^^^
+All MATLAB set commands have the format ``client_object.set("client_entry", “value”)``.
+
+The set function changes the value of the target “client_entry” to value. A value set and not saved will not be retained after a power cycle.
+
+Suppose we want to change the Propeller Motor Controller's timeout parameter to 5 seconds. 
+
+.. image:: ../_static/api_pics/timeout_entry.png
+
+To do so, create a ``PropellerMotorControlClient`` instance ``prop``. Then, run ``prop.set(“timeout”, 5)``. To confirm that the set was successful, we can run a ``get``.
+
+.. image:: ../_static/api_pics/get_timeout.png
+
+Save
+^^^^^^
+All MATLAB save commands have the format ``client_object.save("client_entry")``.
+
+The save function takes the currently set entry value, and stores it in the module's persistent memory. Values that are saved are retained on power cycles.
+
+Suppose we want to save the timeout value set above. To do so, run ``prop.save('timeout')``.
```

Comment:
I know its a bit silly, but popping this out to a code block or picture may be good for consistency/eye-catching